sql/src/main/java/org/opensearch/sql/sql/parser/AstSortBuilder.java

```diff
@@ -57,8 +54,8 @@ private List<Field> createSortFields() {
   }
 
   /**
-   * Argument "asc" is required.
-   * Argument "nullFirst" is optional and determined by Analyzer later if absent.
+   * Argument "asc" is required. Argument "nullFirst" is optional and determined by Analyzer later
```

Comment:
I think this comment is outdated and can be removed.

Reply:
Are you addressing the entire comment?

Reply:
You can fix it or leave it.  The change request is unrelated to your work.